website/source/guides/governance-and-policy/quotas.html.md

```diff
@@ -65,13 +65,17 @@ limit {
     region_limit {
         cpu = 2500
         memory = 1000
+		networks = [
+			{ mbits = 50 }
+		]
     }
 }
 ```
 
 A quota specification is composed of one or more resource limits. Each limit
 applies to a particular Nomad region. Within the limit object, operators can
-specify the allowed CPU and memory usage.
+specify the allowed CPU, memory usage, and network bandwidth. Network limits may
+only specify a single total limit for bandwidth.
```

Comment:
Should add language here that network mbits for quota were introduced in 0.10.2, and is optional. If left unspecified Nomad will not enforce network mbits.